apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.js

```diff
@@ -156,31 +161,40 @@ router.get(
       const groupConfig = await groupAssessmentHelper.getGroupConfig(res.locals.assessment.id);
       res.locals.groupConfig = groupConfig;
 
-      // Check whether the user is currently in a group in the current assessment by trying to get a group_id
```

Comment:
Is this check now being performed somewhere else, or was it just not necessary?

Reply:
It was trying to find the group based on the user instead of based on the assessment instance. This was a problem when the user was an instructor. This was changed to use the group from the assessment instance, which is presumed to always exist. This was copied from the assessment code (which needs to check if the group exists) without regard for the fact that the group in fact exists.